tests/tpool_test.py

```diff
@@ -366,7 +366,7 @@ def test_leakage_from_tracebacks(self):
 
 
 def test_isolate_from_socket_default_timeout():
-    tests.run_isolated('tpool_isolate_socket_default_timeout.py', timeout=1)
+    tests.run_isolated('tpool_isolate_socket_default_timeout.py', timeout=5)
```

Comment:
Would it be reasonable to conditionally set 5 here only on SPARC64, since it seems to be working fine elsewhere?

Reply:
I suspected that it could hit the same issue e.g. on ARM, or simply on systems under heavy load (we run tests as part of package installation, so other packages may be compiling in parallel).